client/html/parts/body.html

```diff
@@ -0,0 +1,10 @@
+<body class="menubar-hoverable header-fixed">
```

Comment:
It wasn't missing, it was only in another file. I decided to move it to parts/ because now we have body, header, head, menubar, offcanvas only in one place, these are all htmls that are present in almost all routes, so better put them together to better keep track of them